can4docker/gateway.py

```diff
@@ -22,12 +16,14 @@ def add_rule(self, src_netdev, dst_netdev):
         if (not self.rules.get(src_netdev, None)):
             self.rules[src_netdev] = []
         if not dst_netdev in self.rules[src_netdev]:
-            LOGGER.debug("/Gateway.add_rule: Adding rule for {} to {}".format(src_netdev, dst_netdev))
+            LOGGER.debug("/Gateway.add_rule: Adding rules for {} to {}".format(src_netdev, dst_netdev))
             utils.sh("cangw -A -s {} -d {} -eX".format(src_netdev, dst_netdev))
+            utils.sh("cangw -A -s {} -d {} -e".format(src_netdev, dst_netdev))
```

Comment:
This and the added line below are the significant changes in the PR.  The rest is white space and typo cleanup.